### PR TITLE
Add how-to instruction for view-only mode

### PR DESCRIPTION
**Pull request by xuesichao**
**Issue #:**
Need to add how-to doc for view-only mode.

**Description of changes:**
Add how-to instruction for view-only mode.

**Testing**
1. Have you successfully run `npm run build:release` locally?
Yes
2. How did you test these changes? 
N/A
3. Can these changes be tested using the demo application? If yes, which demo application can be used to test it?
N/A
4. Do you add, modify, or delete public API definitions? If yes, has that been reviewed and approved?
N/A
5. Do you change the wire protocol, e.g. the request method? If yes, has that been reviewed and approved?
N/A

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: guides/03_API_Overview.md [hunk 1]
```diff
@@ -48,6 +48,34 @@ When obtaining devices to configure, the browser may initially decline to provid
 
 You can override the behavior of the device-label trigger by calling meetingSession.audioVideo.[setDeviceLabelTrigger(trigger)](https://aws.github.io/amazon-chime-sdk-js/interfaces/audiovideofacade.html#setdevicelabeltrigger).
 
+#### Implement a view-only/observer/spectator experience
+
+We recommend that you use `deviceLabelsTrigger` to control device permissions.
```
Comment (vidya-mohan) [inline guides/03_API_Overview.md:53]: Could we introduce the problem in the starting paragraph, "To enable a ...."

Comment (vidya-mohan) [reply guides/03_API_Overview.md:53]: could we also say "the device permission prompts for audio and video"

Comment (vidya-mohan) [reply guides/03_API_Overview.md:53]: Also clarify that by doing so tthey are not impacted in their ability to view content or listen in on audio in their meeting experience

Comment (xuesichao) [reply guides/03_API_Overview.md:53]: How about:
"To enable a view-only mode, you need to control the device permission. We recommend that you implement it based on 'deviceLabelsTrigger`."

Comment (xuesichao) [reply guides/03_API_Overview.md:53]: > Also clarify that by doing so tthey are not impacted in their ability to view content or listen in on audio in their meeting experience
Sure, I will add it to the React Component how-to as well. It is good to mention this.

Comment (xuesichao) [reply guides/03_API_Overview.md]: Updated.

File: CHANGELOG.md [hunk 2]
```diff
@@ -11,23 +11,21 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
...
-
 ## [2.12.0] - 2021-06-23
 
 ### Added
 
 - [Documentation] Add documentation for `getObservableVideoMetrics`.
 - [Documentation] Update FAQ and public documentation to add more information on SignalingBadRequest related error codes.
 - [Documentation] Rephrase the terms in the status code documentations.
-- Add events ingestion to report meeting events to Amazon Chime backend.
-  Check [Client Event Ingestion guide](https://aws.github.io/amazon-chime-sdk-js/modules/clienteventingestion.html) for more information.
```
Comment (xuesichao) [inline CHANGELOG.md:30]: FYI: Remove the duplicate record based on discussion with teamate

**Review by jing-chen1 — APPROVED** [review]

File: guides/03_API_Overview.md [hunk 3]
```diff
@@ -48,6 +48,46 @@ When obtaining devices to configure, the browser may initially decline to provid
...
+const audioOutputDevices = await meetingSession.audioVideo.listAudioOutputDevices();
+const videoInputDevices = await meetingSession.audioVideo.listVideoInputDevices();
+await meetingSession.audioVideo.chooseAudioInputDevice(audioInputDevice.deviceId);
+await meetingSession.audioVideo.chooseAudioOutputDevice(audioOutputDevice.deviceId);
+await meetingSession.audioVideo.chooseVideoInputDevice(videoInputDevices.deviceId);
+```
+
+#### Safari user are not able to join the meeting in view-only mode
+
+Safari users may not be able to successfully join the meeting in view-only mode, due to this know issue [#474](https://github.com/aws/amazon-chime-sdk-js/issues/474). While we are still trying to solve it, you can fix it locally by allowing Safari's Autoplay. The specific steps are:
```
Comment (jing-chen1) [inline guides/03_API_Overview.md:83]: Typo: `known`

Comment (jing-chen1) [reply guides/03_API_Overview.md:83]: Maybe we can say `Since we are still working on this`

File: guides/03_API_Overview.md [hunk 4]
```diff
@@ -48,6 +48,46 @@ When obtaining devices to configure, the browser may initially decline to provid
...
+
+1. Open your application in the Safari app on your Mac.
+2. Choose Safari > Settings for This Website.
```
Comment (jing-chen1) [inline guides/03_API_Overview.md:86]: In my Safari(14.1.1), the Auto-play is under preference.

**Review by jing-chen1 — APPROVED** [review]

File: guides/03_API_Overview.md (hunk 4 above)
Comment (xuesichao) [reply guides/03_API_Overview.md:86]: That option appears only when user opens a website.

**Pull request merged**